### ISSUE TITLE
Run xsum pre-trained BART-large on test set

### ISSUE THREAD

**Issue by dleve123** [dleve123/topics-in-nlp-repro-project#2]
- add linear classification layer

**Comment by dleve123** [comment]
Considering that in the paper, they use the [Xsum-finetined model trained by Facebook](https://github.com/CogComp/faithful_summarization/blob/42b7f79955a68f92961b9ea62f7f144a65375adb/README.md#example-corrections-and-evaluation), I'm rescoping this task to just run inference on the test set and ensure that the ROUGE and BERT scores overlap.
